src/EntryPoints/REST/ApprovePageApi.php

```diff
@@ -66,8 +69,23 @@ private function revisionIsLatest( int $revisionId, WikiPage $page ): bool {
 		return $revisionId === $page->getRevisionRecord()?->getId();
 	}
 
-	public function newSuccessResponse(): Response {
-		return $this->getResponseFactory()->createNoContent();
+	public function newSuccessResponse( ?ApprovalState $state ): Response {
+		if ( $state === null ) {
+			return $this->getResponseFactory()->createNoContent();
+		}
+
+		return $this->getResponseFactory()->createJson( [
+			'approvalTimestamp' => $state->approvalTimestamp,
+			'approver' => $this->getUserNameFromUserId( $state->approverId ),
```

Comment:
I think it should return UserRealName and not the username since I am actually using `user_real_name` from user table and if the real name doesn't exist then perhaps maybe username. @malberts  @JeroenDeDauw